tests/gen-installed-test.py

```diff
@@ -29,7 +29,11 @@ def write_template(filename, data):
         f.write(data)
 
 def build_template(testdir, testname):
-    return "[Test]\nType=session\nExec={}\n".format(os.path.join(testdir, testname))
+    return """[Test]
+Type=session
+Exec={}
+TestEnvironment=MUTEST_OUTPUT=tap
```

Comment:
This is a string list, so it should be:

```suggestion
TestEnvironment=MUTEST_OUTPUT=tap;
```